environment/aws/README.md

```diff
@@ -106,30 +124,17 @@ The Sync Gateway URL and Couchbase Server version properties should be self expl
 - TDK config out: An optional file to write the resulting TDK config file to (otherwise it will go to stdout)
 - Topology is the toplogy JSON file that will describe how to set up AWS instances (see the [topology README](./topology_setup/README.md) for more information.)
 
-:exclamation: It is important to understand how the template system works for the TDK config in.  This system is capable of creating any number of couchbase server and sync gateway instances, and once it does it will begin to replace the following templates with actual IP addresses:  `\{\{cbs-ip\d+\}\}` / `\{\{sgw-ip\d+\}\}`.  So `{{cbs-ip1}}` will receive the IP address of the first created Couchbase Server, and correspondingly `{{sgw-ip1}}` will receive the IP address of the first created Sync Gateway, and so on.  A simple template case would look like this:
-
-```json5
-{
-    //...
-    "sync-gateways": [{"hostname": "{{sgw-ip1}}", "tls": true}],
-    "couchbase-servers": [{"hostname": "{{cbs-ip1}}"}],
-    //...
-}
-```
 
 ### Stopping
 
 ```
-usage: stop_backend.py [-h] --public-key-name PUBLIC_KEY_NAME
+usage: stop_backend.py [-h] [--topology TOPOLOGY]
 
 Tear down a previously created E2E AWS EC2 testing backend
 
 optional arguments:
-  -h, --help            show this help message and exit
-
-required arguments:
-  --public-key-name PUBLIC_KEY_NAME
-                        The public key stored in AWS that pairs with the private key
+  -h, --help           show this help message and exit
+  --topology TOPOLOGY  The topology file that was used to start the environment
 ```
 
-The stop script only has one argument that is required, and is only required as what seems to be a quirk of Terraform.  Since the Terraform config declares a variable to be used, it must be specified in all commands (which includes destroy), so the public key name from step 0 of this README should be used here as well.  
\ No newline at end of file
+The stop script only has one argument which is the topology file that was used to run start_backend above.  If no file was provided, then it should be omitted to use the default.
```

Comment:
Again, quite picky but I had to read this twice to understand what it was saying.  Perhaps a different wording?
"If no topology file was provided for `start_backend` then no file should be provided here, either: the default will be used"